.changeset/eight-kangaroos-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/charts": patch
+---
+
+fix: assigned-cursor
```

Comment:
Can you change the description of `changeset`?
```diff md
-fix: assigned-cursor
+fix: assigned className to cursor
```

Reply:
i did it.

Reply:
Thank you!!